src/Resources/views/storefront/base.html.twig

```diff
@@ -1,11 +1,11 @@
 {% sw_extends '@Parent/storefront/base.html.twig' %}
 
 {% block base_body_inner %}
-  <ff-communication
-    {{ page.extensions.factfinder.communication|map((val, key) => key ~ '="' ~ (val|e('html_attr')) ~ '"')|join(' ')|raw }}
-    only-search-params="true"
-    disable-single-hit-redirect="true"></ff-communication>
-
+  {% set factfinder_config = page.extensions.factfinder.all.communication %}
```

Comment:
Why this change?
With that we do not allow users to add more attributes to `ff-communication`, which element has much more

we also cant set `default-query="shirt"` and `search-immediate` because not everyone is selling clothes and not everyone wants to use `search-immediate` everywhere